src/lucky/remote_ip_address.cr

```diff
@@ -0,0 +1,15 @@
+module Lucky::RemoteIpAddress
+  # Returns a `String` IP Address of the remote client by
+  # looking at a few different possible options.
+  #
+  # * HTTP_X_FORWARDED_FOR header
+  # * REMOTE_ADDRESS header
+  # * Fallback to localhost 127.0.0.1
+  def remote_ip : String
+    request = @context.request
+
+    request.headers["HTTP_X_FORWARDED_FOR"]?.try(&.split(',').first) ||
+      request.remote_address ||
+      "127.0.0.1"
```

Comment:
I think rather than returning a potentially incorrect remote ip it'd be better to use nil and let people handle that however you want. For example if the ip is empty you may want to raise, or maybe log. Right now you couldn't do that because you don't know if the ip was actually 127.0.0.1 or if it just fell back to that